### PR TITLE
Cfy 2090 create influxdb docker image

### PR DESCRIPTION
**Pull request by nir0s**

### CODE REVIEW

File: docker/influxdb/Dockerfile [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
-ENV VERSION=0.8.8 \
+ENV INFLUXDB_VERSION=0.8.8 \
     DOCKER_ENV=True \
     INFLUXDB_SERVICE_DIR=/opt/influxdb
 ENV INFLUXDB_CONFIG_FILE=${INFLUXDB_SERVICE_DIR}/shared/config.toml
 
 RUN apt-get update && \
     apt-get install -y curl && \
-    curl http://s3.amazonaws.com/influxdb/influxdb_${VERSION}_amd64.deb --create-dirs -o /opt/tmp/influxdb.deb && \
+    curl http://s3.amazonaws.com/influxdb/influxdb_${INFLUXDB_VERSION}_amd64.deb --create-dirs -o /opt/tmp/influxdb.deb && \
```
Comment (m1keil) [inline docker/influxdb/Dockerfile]: put it on /tmp
no need to create dirs and stuff

File: docker/influxdb/Dockerfile [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
     dpkg -i /opt/tmp/influxdb.deb && \
     # starting influxdb as daemon to create cloudify db
-    /bin/bash -c 'source /opt/tmp/utils.sh && \
     /usr/bin/influxdb-daemon -config=${INFLUXDB_CONFIG_FILE} && \
-    wait_for_port 8086' && \
+    chmod +x /opt/scripts/wait_for_port && sleep 3 && /opt/scripts/wait_for_port 8086 && \
```
Comment (m1keil) [inline docker/influxdb/Dockerfile]: no need to +x as well
just run with sh or bash

File: docker/influxdb/Dockerfile [hunk 3]
```diff
@@ -1,25 +1,23 @@
...
     curl -s "http://localhost:8086/db?u=root&p=root" -d "{\"name\": \"cloudify\"}" && \
-    rm -rf /var/lib/apt/lists; rm /opt/tmp/*; apt-get remove curl -y; apt-get autoremove -y
+    rm -rf /var/lib/apt/lists; rm /opt/tmp/*; apt-get purge curl -y; apt-get autoremove -y
```
Comment (m1keil) [inline docker/influxdb/Dockerfile]: don't forger to change to /tmp/*

File: docker/influxdb/Dockerfile (hunk 1 above)
Comment (cosmo-admin) [inline docker/influxdb/Dockerfile]: oops

**Pull request merged**